people.md

```diff
@@ -27,6 +27,18 @@ Alphabetical by first name, names are followed by GitHub usernames.
 | JupyterHub and Binder | Min Ragan-Kelley | [@minrk](https://github.com/minrk) |
 | Voilà | Martin Renou | [@martinRenou](https://github.com/martinRenou) |
 
+## [Jupyter Foundation](jupyter_foundation.md) Governing Board
```

Comment:
To be filled out with other members over time. But we can start with the EC right now.

Reply:
:+1: